js/index.js

```diff
@@ -11,6 +11,7 @@ copyright.innerHTML  = "&copy; Lawrence " + thisYear;
 footer.appendChild(copyright);
 
 let skills = ["Javascript", "HTML", "CSS", "Java", "Python", "C", "Git", "a little bit of Ruby"];
```

Comment:
Nothing wrong with any of this code -- just a thought that if you never reuse a variable, it might be helpful to default to `const` instead of `let` to indicate to your future-self that when you wrote this code, you did not intend the variable to be altered.